C/tests/cmake/platform_linux.cmake

```diff
@@ -33,5 +33,6 @@ function(setup_build)
         ${LIBCXXABI_LIB}
         ${ZLIB_LIB}
         dl
+        rt
```

Comment:
What is "rt".

Reply:
Yes, it is the library for [POSIX.1b realtime extensions](https://www.unix.com/man-page/sunos/3lib/librt/)